### ISSUE TITLE
Post controls truncated on iPhone SE 1st-gen when instance has large character limit

### ISSUE THREAD

**Issue by VyrCossont** [feditext/feditext#214]
**To Reproduce**
1. Be on an instance with a 5-digit character limit.
2. Start a new post.
3. The (+) button on the post controls bar is shrunken nigh unto invisibility, and the button for adding photos is gone.
4. See screenshot.

**Expected behavior**
The (+) button for adding a new post to the thread should be visible. 

**Screenshots**
![Feditext screenshot showing tiny plus button](https://github.com/feditext/feditext/assets/26195552/60f77b2b-8b07-4c4e-8e32-837b478e1c80)

**(please complete the following information):**
 - Device: iPhone SE (1st generation)
 - OS: 15.5
 - App Version: 1.7.2 (build 70)

**Additional context**
The workaround is to post one post at a time (for threads) or to copy and paste media into the post. Both are annoying.

Fixing this involves overhauling the post controls input accessory view, which we need to do for several other reasons, so I'm punting it to 1.8.

**Comment by VyrCossont** [comment]
Not deterministic: closing the new post and starting another one shrinks the emoji picker to unusable size instead.

![photo button visible, emoji button shrunk, new post button missing](https://github.com/feditext/feditext/assets/26195552/4a2ede7e-68cb-4bc6-b67b-a9f22922629d)

**Comment by VyrCossont** [comment]
Impact is, of course, worse at large text sizes.

**Comment by VyrCossont** [comment]
(accidental close)

**Comment by mirabilos** [comment]
oh, that’s supposed to be an emoji picker ;) I normally use the emoji keyboard for them, but yeah, this matches what I noticed, plus you can still hit the white area to the left where the bureau clip is supposed to be and it’ll still allow attaching photos
